src/encoded/static/components/publication.js

```diff
@@ -42,17 +48,179 @@ const datasetsColumns = {
     lab: {
         getValue: dataset => (dataset.lab ? dataset.lab.title : ''),
         title: 'Lab',
+        sorter: false,
     },
     biosample_summary: {
         title: 'Biosample Summary',
+        sorter: false,
     },
     status: {
-        display: datasets => <Status item={datasets.status} badgeSize="small" css="result-table__status" />,
+        display: dataset => <Status item={dataset.status} badgeSize="small" inline />,
         title: 'Status',
+        sorter: false,
+    },
+    cart: {
+        title: 'Cart',
+        display: dataset => <CartToggle element={dataset} />,
+        sorter: false,
+    },
+};
+
+
+const publicationDataColumns = {
+    accession: {
+        display: publicationData => <a href={publicationData['@id']}>{publicationData.accession}</a>,
+        title: 'Accession',
+        sorter: false,
     },
+    fileCount: {
+        getValue: publicationData => (publicationData.files ? publicationData.files.length : 0),
+        title: 'Number of files',
+        sorter: false,
+    },
+    description: {
+        display: publicationData => publicationData.description || <i>No description</i>,
+        title: 'Description',
+        sorter: false,
+    },
+    status: {
+        display: publicationData => <Status item={publicationData} badgeSize="small" inline />,
+        title: 'Status',
+        sorter: false,
+    },
+};
```

Comment:
It would be sort of handy maybe to be able to specify that none of the columns are sortable in cases like these, just to make this less verbose, but I don't think that would be trivial and this works well.

Reply:
Yeah, could be a new prop on SortTable I suppose.